src/main/webapp/app/faq/faq.component.ts

```
@@ -24,11 +27,14 @@ import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
     imports: [ArtemisSharedModule, CustomExerciseCategoryBadgeComponent, ArtemisSharedComponentModule, ArtemisMarkdownModule],
 })
 export class FaqComponent implements OnInit, OnDestroy {
+    protected readonly FaqState = FaqState;
     faqs: Faq[];
+    course: Course;
     filteredFaqs: Faq[];
     existingCategories: FaqCategory[];
     courseId: number;
     hasCategories: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)